protostar-rust/tests/data/declare_test/Scarb.toml

```diff
@@ -6,6 +6,6 @@ version = "0.1.0"
 sierra = true
 
 [dependencies]
-starknet = ">=1.1.0"
+starknet = "2.0.0-rc3"
```

Comment:
Can we add TODO + issue so we remember that we need to resolve version problems?

Reply:
why is it a problem exactly?

Reply:
We have cairo fork version 2.0.0-rc2 but Scarb 0.5.0-alpha.1 uses v2.0.0-rc3. Maybe downgrading Scarb to 0.5.0-alpha.1 is a solution?

Reply:
Changed so we use rc2 everywhere.